pyuvdata/uvcal/tests/test_uvcal.py

```diff
@@ -4519,7 +4497,8 @@ def test_init_from_uvdata_freqrange_errors(uvcalibrate_data):
 
     with pytest.raises(
         ValueError,
-        match="if future_array_shapes is True, freq_range must be an array shaped like",
+        match="UVParameter \\_freq\\_range is not expected shape\\. Parameter shape is "
+        "\\(1\\, 4\\)\\, expected shape is \\(1\\, 2\\)\\.",
     ):
```

Comment:
I think you can avoid using all the `\\`s here and below if you just call `re.escape` on the whole string.